pmm-tests/pmm-2-0-bats-tests/pmm-testsuite.sh

```diff
@@ -232,6 +241,8 @@ fi
 if [[ $instance_t == "ps" ]]; then
   echo "Running PS specific tests"
   run_ps_specific_tests
+  echo "Running MySQL TLS tests"
```

Comment:
I realized this would call the tls tests 2 times with 5.7 and 8.0 both, can you move this call post the pxc instance tests, https://github.com/percona/pmm-qa/blob/91d269ce1177cc518d1f6a6c2bf92e53876f23fe/pmm-tests/pmm-2-0-bats-tests/pmm-testsuite.sh#L261 post the docker-env checks we run

Reply:
OK :)